### DISCUSSION TITLE
Should clifm open links as normal files?

### DISCUSSION THREAD

**Discussion by tkossak** [leo-arch/clifm#205] [Q&A] [answered]
When I try to edit text file I just write it's name or ELN in command line, but it doesn't work for links eg:

```
<0> $ ls -l .xprofile
lrwxrwxrwx 1 kossak kossak 44 03-26 13:50 .xprofile -> /home/kossak/apps/dotfiles/dotfiles/xprofile
T11[1] 14:23 kossak:lman3 ~
<0> $ .xprofile
open: .xprofile ((null)): Cannot open file
Try 'APP FILE' or 'open FILE APP'
T11[1] 14:23 kossak:lman3 ~
<1> $ 112
open: .xprofile ((null)): Cannot open file
Try 'APP FILE' or 'open FILE APP'
T11[1] 14:23 kossak:lman3 ~
<1> $ cat 112
xset -b
T11[1] 14:23 kossak:lman3 ~
<0> $
```

Is it possible to make clifm open links in the same way as normal files?

**Comment by leo-arch** [comment, marked as answer]
Hi @tkossak. Yes, you're right. I'll take a look at it and let you know.

Thanks for reporting.

**Reply by leo-arch** [reply]
You spotted an important bug! Thank you very much!

It should be solved now.

**Reply by tkossak** [reply]
Thank you very much, it works now.
